superset-frontend/packages/superset-ui-core/src/query/types/Dashboard.ts

```diff
@@ -80,6 +80,24 @@ export type Filter = {
   description: string;
 };
 
+export type AppliedFilter = {
+  values: {
+    filters: Record<string, any>[];
+  } | null;
+};
+
+export type AppliedCrossFilterType = {
+  filterType: undefined;
+  targets: number[];
+  scope: number[];
+} & AppliedFilter;
+
+export type AppliedNativeFilterType = {
+  filterType: 'filter_select';
+  scope?: number[] | NativeFilterScope; // TODO: inconsistent usage
+  targets: [Partial<NativeFilterTarget>];
```

Comment:
Do you intent to have this?

```suggestion
  targets: Partial<NativeFilterTarget>[];
```